docs/admin/networking/index.md

```diff
@@ -197,6 +197,64 @@ low-latency browser experiences for geo-distributed teams.
 
 To learn more, see [Workspace Proxies](./workspace-proxies.md).
 
+## Latency
+
+Coder measures and reports several types of latency, providing insights into the performance of your deployment. Understanding these metrics can help you diagnose issues and optimize the user experience.
+
+There are three main types of latency metrics for your Coder deployment:
+
+- Dashboard-to-server latency:
+
+  The Coder UI measures round-trip time to the Coder server or workspace proxy using built-in browser timing capabilities.
+
+  This appears in the user interface next to your username, showing how responsive the dashboard is.
+
+- Workspace connection latency:
+
+  The latency shown on the workspace dashboard measures the round-trip time between the workspace agent and its DERP relay server.
+
+  This metric is displayed in milliseconds on the workspace dashboard and specifically shows the agent-to-relay latency, not direct P2P connections.
+
+  Total end-to-end latency experienced by a user is a combination of dashboard-to-server latency plus this agent-to-relay latency.
+
+- Database latency:
+
+  For administrators, Coder monitors and reports database query performance in the health dashboard.
```

Comment:
> Ok, I've looked into this, and admittedly I'm a bit embarassed by what we built.
> 
> The latency on the Agent resource card is between the Agent and the shown relay. It never shows P2P (which isn't possible to achieve via the web browser anyway).
> 
> The estimated total latency if you were to access a workspace via the browser would be if you added the numbers by your username to the numbers you see in the Agent card, then that would give you a rough picture of the total latency thru various relays.

@spikecurtis copying your comments over so they don't get lost

I removed the lines you had commented on, and replaced them with this. Do you think it helps to point out `not direct P2P connections` or should we cut that?